cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -651,6 +790,58 @@
     "**IN PROGRESS: If a reference other than one of the default CellRanger reference files was used, a map file can be created from the annotation file for use in curation**"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**If a GENCODE/ENSEMBL reference was used, parse the annotation file**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use pd.read_table to parse the gtf & apply various manipulations to produce symbol-ENSG table\n",
+    "gtf = pd.read_table('path/to/gtf', header=None)\n",
```

Comment:
I think it will be more user-friendly to define the 'path/to/gtf' separated from the rest of the code in the line. So...
gtf_file = 'path/to/gtf'
gtf = pd.read_table(gtf_file, header=None)

but trying on a gencode file, I get
`ParserError: Error tokenizing data. C error: Expected 1 fields in line 6, saw 9`
from this read_table

I think `comment='#'` parameter is needed

Reply:
i believe i've fixed this: used more regex patterns/contain function -- see extract_gene_id_and symb()